miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleService.java

```diff
@@ -381,7 +381,6 @@ private DetailedSample findOrCreateParent(DetailedSample sample) throws IOExcept
   private SampleTissue createParentTissue(SampleTissue tissue, DetailedSample child) throws IOException {
     log.debug("Creating a new Tissue to use as a parent.");
     tissue.setProject(child.getProject());
-    tissue.setDescription("Tissue");
```

Comment:
I will make the ticket. I agree it's not the solution, or even a good solution, but I think adding "ghost" to the description is a user-friendly stopgap until we ultimately make better use of the 'isSynthetic' field.